package.json

```diff
@@ -47,6 +47,8 @@
     "framer-motion": "^9.0.3",
     "gh-pages": "^5.0.0",
     "html-to-image": "^1.11.11",
+    "patch-package": "^7.0.0",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
Did you installed this package for [this reason](https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn)? I'd be glad if you write more specific information when you add external libraries that why we really need this :)